### ISSUE TITLE
Attacking and selecting monsters larger than 1x1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#8]
```
When spawning monsters larger than 1x1, the attack selection position and 
attack animation is not working as expected. 

Enable debug resources (in the code) and use the spawn button to see the bug.
```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 20 Nov 2010 at 10:03

**Comment by GoogleCodeExporter** [comment]
```
Hm... I thought I've played all through 0.6.6 wherever possible, but I haven't 
encountered anything larger than 1x1 :)
```

Original comment by `surrano` on 8 Dec 2010 at 10:55

**Comment by GoogleCodeExporter** [comment]
```
There is no 2x2 monster in the game yet.
IIRC Oskar did mention somewhere that you can currently spawn 2x2 monsters only 
in debug mode. (you need to recompile the game.)
```

Original comment by `SamuelPl...@gmail.com` on 8 Dec 2010 at 11:44

**Comment by GoogleCodeExporter** [comment]
```
This is now fixed for v0.6.7.
```

Original comment by `oskar.wi...@gmail.com` on 18 Dec 2010 at 9:25
- Changed state: **Ready**
- Added labels: **Milestone-0.6.7**

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 20 Dec 2010 at 9:09
- Changed state: **Fixed**
